app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,32 @@
+class RecipeFoodsController < ApplicationController
```

Comment:
- [ ] Please check my comment in the recording , 

 
https://user-images.githubusercontent.com/75126481/219119527-90ab9fc7-e77a-414d-af8c-51307c19c900.mov


## This prevents me from seeing other functionalities so for the next reviewer will check the other functionalities and may require changes , Please double check before you request for another review thanks 😉

Reply:
Thanks for the review. We have double checked on both our local devices and it works. This is a link to show how it works as well. [https://www.loom.com/share/8310ac3706254696bfef6fe584c74812](project)